terraform/application/application.tf

```diff
@@ -17,9 +17,12 @@ module "application_configuration" {
       PGSSLMODE          = local.postgres_ssl_mode
       CANONICAL_HOSTNAME = local.canonical_hostname
   })
-  secret_variables = {
-    DATABASE_URL = module.postgres.url
-  }
+  secret_variables = merge(
+    {
+      DATABASE_URL         = module.postgres.url
+      HEARTBEAT_CHECK_URLS = join(",", values(module.statuscake[0].heartbeat_check_urls))
```

Comment:
We'll most probably need only 1 check, so something like this would be enough:
```
HEARTBEAT_CHECK_URL = module.statuscake[0].heartbeat_check_urls[0]
```